### DISCUSSION TITLE
The easiest way to handle sessions/connexions

### DISCUSSION THREAD

**Discussion by ZiRaymondal** [zeroc-ice/ice-demos#137] [Q&A] [answered]
Hello ! Hope you're doing well !

I need to identify connexions/sessions without auth system or whatever complicated.  
I was thinking about generating an UUID client side which is stored and sent to Server once the co established in order to identify by a String or Long Number connected devices to an interface.  
But I know that this method is kinda dirty and will probably be difficult to manage server side when exceptions occurs like a broken co.  

I was wondering if there was a clean & fast to do that with Ice Zeroc.  
Have a good day!

**Comment by pepone** [comment, marked as answer]
Hi,

You should check the [Ice/interceptor](https://github.com/zeroc-ice/ice-demos/tree/3.7/cpp11/Ice/interceptor) demo it does something similar to what you describe using the request context and invocation interceptors.

Cheers,
Jose
